cmake/scripts/CreateGRF.cmake

```
@@ -55,4 +58,18 @@ if(RESULT)
     message(FATAL_ERROR "GRFCodec failed")
 endif()
 
+execute_process(COMMAND ${GRFID_EXECUTABLE} -m ${GRF_SOURCE_FOLDER_NAME}.grf OUTPUT_VARIABLE GRFID_HASH RESULT_VARIABLE RESULT)
+if(RESULT)
+    if(NOT RESULT MATCHES "^[0-9]*$")
+        message(FATAL_ERROR "Failed to run GRFID (${RESULT}), please check GRFID_EXECUTABLE variable")
+    endif()
+    message(FATAL_ERROR "GRFID failed")
+endif()
+
+# Trim hash from grfid
+string(STRIP ${GRFID_HASH} GRFID_HASH)
+file(WRITE ${CMAKE_SOURCE_DIR}/${GRF_SOURCE_FOLDER_NAME}.grf.hash ${GRFID_HASH})
```

`${CMAKE_SOURCE_DIR}` is current working dir for script ran with `cmake -P`
But 
```suggestion
file(WRITE ${GRF_BINARY_FILE}.hash ${GRFID_HASH})
```
should work without needing the manual copy.

---

Oops, that was me trying things that didn't do anything, and not reverting it :)